### DISCUSSION TITLE
How to change the dashboard Icon ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6346] [Help]
Hi All,

How to change the dashboard Icon ?

![image](https://user-images.githubusercontent.com/6214964/235181157-228996d4-8fe8-470c-a330-0c79d8cb618a.png)

![image](https://user-images.githubusercontent.com/6214964/235181389-b50d6c5d-c889-4e50-abd3-1129053ea9b1.png)

thanks

**Comment by danharrin** [comment]
Create a new Dashboard page that extends the original Dashboard, set `$navigationIcon`, then replace the old one with the new one in the config file
